scripts/utils/generate_strings.py

```diff
@@ -58,7 +58,7 @@ def parseTranslationStrings(project, yamlfile):
 
         for category in yaml_content:
             for key in yaml_content[category]:
-                string_id = f"{project}.{category}.{key}"
```

Comment:
This is going to break every single translation we have, and duplicate content until we have branches that use two different naming conventions.

Check the string IDs in the XLIFF file
https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/blob/main/en/mozillavpn.xliff#L827

The string IDs should NOT change as part of this, they didn't change as part of the previous move.

Reply:
Brought the `vpn.` prefix back.